storage/Devices/LvmLvImpl.cc

```diff
@@ -120,7 +120,8 @@ namespace storage
     bool
     LvmLv::Impl::is_usable_as_blk_device() const
     {
-	return lv_type == LvType::NORMAL || lv_type == LvType::THIN || lv_type == LvType::RAID;
+	return lv_type == LvType::NORMAL || lv_type == LvType::THIN || lv_type == LvType::RAID ||
```

Comment:
This would be a good candidate for a `switch` and enumerating all possible values; that way the compiler would complain if a new enum value is added, yet forgotten here.

```C++
switch (lv_type)
{
    case LvType:NORMAL:
    case LvType::THIN:
    ...
        return true;

    case LvType::WHATEVER:
    case LvType::WHATNOT:
        return false;
}
```